### DISCUSSION TITLE
Where are settings stored?

### DISCUSSION THREAD

**Discussion by Skiddz** [OrcaSlicer/OrcaSlicer#1594] [General]
Something got seriously buggered with my initial install and I want to start over. I've removed the OrcaSlicer folders from ~/AppData/Local and ~/AppData/Roaming, removed all the registry entries and deleted the entire /Program Files/OrcaSlicer folder.

After unzipping and launching the slicer, it still says it's loading settings. Where are the settings stored?? How can I force the setup wizard to run again?

**Comment by SoftFever** [comment]
@Skiddz 
It's stored here: %appdata%/orcaslicer/

**Reply by Skiddz** [reply]
I must be missing something.  I've deleted all the OrcaSlicer folders from the %appdata%/ tree and it's still loading settings.  I have searched my entire system for "OracSlicer" and am only getting results for the .zip file I downloaded a few days ago.  No "OrcaSlicer" entries in the registry either.

**Reply by SoftFever** [reply]
Hmmm do you see `BambuStudio-softfever` folder there?
Can you try to delete that folder too if there is.

**Reply by Skiddz** [reply]
Nope.  Nothing 'Bambu' or 'softfever' on the system.

Just for giggles, I'll delete the .zip file and download it again.  I *shouldn't* matter, but at this point, I'm scratching my head.

**Reply by SoftFever** [reply]
I assume you are using Windows, right?
Orca only store settings in `C:\Users\your_name\AppData\Roaming\OrcaSlicer`

**Reply by Skiddz** [reply]
Yes, Windows 10 Pro.

OK, I *SERIOUSLY* doubt a new .ZIP file did anything, but I just 'reinstalled' and it went into the setup routine again.  Looks like I'm back in biz once I set up my printers.

Weird....

**Reply by Apie56** [reply]
> @Skiddz It's stored here: %appdata%/orcaslicer/

where can I find project process files? They are not in the same location as the default process/machine/material files.

**Reply by LJKraus** [reply]
Default presets  are in C:\Users\%USERNAME%\AppData\Roaming\OrcaSlicer\system as .json files
User presets are in C:\Users\%USERNAME%\AppData\Roaming\OrcaSlicer\user\ (some number) \ as .json files

I believe presets saved within a project are stored within the .3mf file, as \Metadata\project_settings.config

**Reply by adrutu** [reply]
> I assume you are using Windows, right? Orca only store settings in `C:\Users\your_name\AppData\Roaming\OrcaSlicer`

Thank dude! swapped the old configs over the new ones and its all there, just works. thank you

**Comment by petjek** [comment]
Same issue here but on Mac OS. I removed OrcaSlicer from Programms folder and checked /username/Library for entries but couldn't find a folder OrcaSlicer.

**Comment by webhive** [comment]
Just discovered where are settings stored on macOS

`/Users/<user name>/Library/Application Support/OrcaSlicer/`

**Reply by luchfilip** [reply]
my hero

**Comment by matt-cmd-github** [comment]
Here are where the OrcaSlicer settings are stored for Ubuntu:

`/home/<user name>/.config/OrcaSlicer`

**Comment by einencool** [comment]
Hello, I'm new to OrcaSlicer and used PrusaSlicer the last years.
I've several PCs with windows and use a portable Version of PrusaSlicer and in the "Program Start Shortcut", I've the path for the settings file. 

Is there a way to store the Settings in another Folder than in "C:\.....\?
Would be great to hear from you :-)

**Comment by LJKraus** [comment]
I don't think the configuration folder can be changed from the default shown in Help>Show configuration folder.

At one time you could start the program from the command line and specify a data directory, but I think the command line options were removed around [December last year](https://github.com/SoftFever/OrcaSlicer/issues/3132).   At least, --H (for help) definitely no longer works and I see no results when trying to use the [old options](https://github.com/SoftFever/OrcaSlicer/discussions/1603).

Bambu users can sync settings Studio and OrcaSlicer settings to multiple computers through the Bambu cloud.

**Reply by einencool** [reply]
Thank you for your fast response.

I've seen a video, and when I understand it correctly, then I'll be able to sync between different windows computers when I'm logged in with a Orca account.
Is this correct?

I've got a new printer and so I'll give it a try to start with a more powerful software :)

**Comment by LJKraus** [comment]
My post yesterday seems to be incorrect.

From the release notes for [v2.2.0-beta2](https://github.com/SoftFever/OrcaSlicer/releases/tag/v2.2.0-beta2):
 "If there's a folder named data_dir under the application's path, Orca will use it as the data directory."
 After creating data_dir in the portable version's folder, a  brief test using [v2.2.0-rc](https://github.com/SoftFever/OrcaSlicer/releases/tag/v2.2.0-rc) worked as expected.

The beta2 release notes also refer to the [pull request](https://github.com/SoftFever/OrcaSlicer/pull/6780) for the feature, which in turn indicate that using the --datadir command line option should still work, which would allow placing the configuration folder anywhere.  I may have used it incorrectly yesterday, but will not be testing it further as I have no real need for that option.

**Reply by einencool** [reply]
Great, thank you for this update :-)
I‘ll try it hopefully on sunday.
I have the software on an owncloud server and for the Prusa Software I‘d the data also stored in the same directory, so I hope it will automatically use this option, so I can use the portable Software stored in one place on different PCs.

Thank you and have a nice weekend.

**Comment by djusHa** [comment]
Flatpak config Folder location:
`
/home/<username>/.var/app/io.github.softfever.OrcaSlicer/config/OrcaSlicer
`
